### PR TITLE
Mtn Dew is better

### PR DESCRIPTION
**Pull request by bradenpotter12**

### CODE REVIEW

File: psychedelicPotters.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ This is Braden
 
 No this is Aaron
 
-Pepsi Zero Sugar has zero sugar
```
Comment (bradenpotter12) [inline psychedelicPotters.txt:13]: Should we do mountain dew or pepsi?

File: psychedelicPotters.txt [hunk 2]
```diff
@@ -10,7 +10,7 @@ This is Braden
...
+Mtn Dew Zero Sugar has zero sugar
 
 This is the coolest branch
```
Comment (samgines) [inline psychedelicPotters.txt:15]: do mtn do, its better

Comment (bradenpotter12) [reply psychedelicPotters.txt:15]: Nice!!

**Review by aaronjpotter93 — APPROVED** [review]

**Comment by aaronjpotter93** [comment]
cool I love mountain dew

**Review by samgines — APPROVED** [review]

**Pull request merged**